notebooks/automatic_embedding_tei_inference_endpoints.ipynb

```diff
@@ -0,0 +1,845 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    MAX_WORKERS = 5  # This is for how many async workers you want. Choose based on the model and HW
</pre><p>What is HW?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>